relbench/data/table.py

```diff
@@ -104,6 +104,15 @@ def upto(self, time_stamp: pd.Timestamp) -> Self:
         if self.time_col is None:
             return self
 
+        is_naive_timestamp = time_stamp.tz is None
+        is_naive_time_col = self.df[self.time_col].dt.tz is None
```

Comment:
Don't understand the logic. What is native

Reply:
it means there's no timezone. If you don't want the logic to handle the timezones, then I can modify the logic while creating datasets

Reply:
I think that's better.